### ISSUE TITLE
Problem entering text for more than one gloss prior to time alignment

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/SignStream3#632]
As I recall, this had been a problem some years ago, and it was fixed.  It is back.

I type in the text for two glosses that occur in sequence.  I leave plenty of space between the glosses.
When I time-align the first, however, it comes out overlapped with the second, without pushing the 
second one until a later time (as happened when this got fixed the last time).

<img width="393" alt="Screen Shot 2023-02-02 at 11 19 32 PM" src="https://user-images.githubusercontent.com/13629122/216512290-a04b00a6-41d8-4451-b976-912404804d61.png">

<img width="350" alt="Screen Shot 2023-02-02 at 11 18 13 PM" src="https://user-images.githubusercontent.com/13629122/216512343-0ef4c984-6caf-42f5-94b3-fd567a098989.png">

**Comment by cneidle** [comment]
Found previous ticket:  #306

**Comment by gregorydimitriadis** [comment]
Reproduced.

**Comment by cneidle** [comment]
I don't agree that this is severity low.  FWIW.  #665 illustrates the consequences of this...

**Comment by gregorydimitriadis** [comment]
Current issue with this might be in underlying code having to deal with the case of many text events jutting up against a gloss event, with not enough space for all the text events to display. Need to investigate if this is still the case resulting in the way it behaves, and if there is an alternative method.
